content/applications/general/voip/onsip.rst

```diff
@@ -5,94 +5,141 @@ Use VoIP services in Odoo with OnSIP
 
-- **SIP Login / Browser's Extension**: the OnSIP 'Username'
-- **OnSIP authorization User**: the OnSIP 'Auth Username'
-- **SIP Password**: the OnSIP 'SIP Password'
-- **Handset Extension**: the OnSIP 'Extension'
+Next, the user will need to be setup in Odoo. Every user that will be associated with an OnSIP User
+must also be configured in the Odoo user's settings/preferences.
 
-You can find all this information by logging in at https://admin.onsip.com/users, then select the
-user you want to configure and refer to the fields as pictured below.
+First, navigate to :menuselection:`Settings --> Manage Users --> Select the User`. Once on the user
+that the OnSIP account should be configured with, click on the :guilabel:`Preferences` tab and
+scroll to :guilabel:`VoIP Configuration`. In this section are the fields that need to be filled in
+with On SIP credentials.
 
-.. image:: onsip/onsip03.png
+.. tip::
+   To access the OnSIP credentials, navigate to `OnSIP <https://www.onsip.com/>`_ and log in. An
+   administrative dashboard will appear. In the left menu, click on :guilabel:`Users` and then
+   select the user. By default the user will open on the :guilabel:`User Info` tab. Click on the
+   :guilabel:`Phone Settings` tab to reveal  OnSIP configuration credentials. In the first column,
+   last in the list of credentials.
+
+Fill in the following fields with the associated credentials listed below:
+
+- :guilabel:`SIP Login` / :guilabel:`Browser's Extension` = OnSIP :guilabel:`Username`
+- :guilabel:`OnSIP authorization User` = OnSIP :guilabel:`Auth Username`
+- :guilabel:`SIP Password` = OnSIP :guilabel:`SIP Password`
+- :guilabel:`Handset Extension` = OnSIP :guilabel:`Ext.` (extension without the `x`)
+
+.. image:: onsip/onsip-creds.png
    :align: center
+   :alt: Onsip user credentials with username, auth username, SIP password, and extension
+         highlighted.
+
+.. tip::
+   The OnSIP extension can be found in the :guilabel:`User` banner line above the tabs.
+
+Once these steps are complete, and the work is saved, Odoo users can make phone calls by clicking
+:guilabel:`📞 (phone)` icon  in the top right corner of Odoo.
 
-You can now make phone calls by clicking the phone icon in the top right corner of Odoo (make sure
-you are logged in as a user properly configured in Odoo and in OnSIP).
+.. seealso::
+   Additional setup and troubleshooting steps can be found on `OnSIP's knowledgebase
+   <https://support.onsip.com/hc/en-us>`_.
 
-.. note::
-   If you see a *Missing Parameters* message in the Odoo softphone, make sure to refresh your Odoo
-   window and try again.
+Incoming calls
+--------------
+
+The Odoo database will also receive incoming calls that produce pop-up windows in Odoo. Click the
+green :guilabel:`📞 (phone)` icon to answer the call.
+
+   .. image:: onsip/incoming-call.png
+      :align: center
+      :alt: alt text
+
+.. seealso::
+   :doc:`voip_widget`
+
+Troubleshooting
+---------------
+
+.. tip::
```

Comment:
I don't think you need to block out these sections as "tips" since they both fall under the Troubleshooting heading. I think it would make it easier to navigate the doc if you added subheadings for each tip instead of separating them out with admonition blocks.